.github/workflows/ci.yml

```diff
@@ -28,5 +28,8 @@ jobs:
           go-version: ${{ matrix.go }}
       - run: make deps
       - run: make lint
+      - run: make install
       - run: make test
       - run: make build
+      - run: make test-examples
+        if: matrix.os != 'windows-latest'
```

Comment:
Do you have an explanation for Windows failure ?

Reply:
nope